lib/protobuf/protoc/generator/service.ex

```diff
@@ -20,12 +30,15 @@ defmodule Protobuf.Protoc.Generator.Service do
     name = Util.attach_raw_pkg(desc.name, ctx.package)
     methods = Enum.map(desc.method, &generate_service_method(ctx, &1))
     generate_desc = if ctx.gen_descriptors?, do: desc, else: nil
-    Protobuf.Protoc.Template.service(mod_name, name, methods, generate_desc)
+    service_template(mod_name, name, methods, generate_desc)
   end
 
   defp generate_service_method(ctx, method) do
     input = service_arg(Util.type_from_type_name(ctx, method.input_type), method.client_streaming)
-    output = service_arg(Util.type_from_type_name(ctx, method.output_type), method.server_streaming)
+
+    output =
```

Comment:
Can't spot the diff here. Maybe a formatter issue?

Reply:
Yep, formatter